components/layout/footer/social-media.tsx

```diff
@@ -0,0 +1,107 @@
+import { StorageType, SocialMedia } from "@/types/storage";
+import { UiContent } from "@/types/ui-content";
+import Image from "next/image";
+//TODO: Change png icons to svg (facebook, instagram, twitter, youtube...)
+//TODO: Map social media in uiContent and display them with their icons
+//TODO: Place that type to the appropiate place in the code
```

Comment:
please read these TODO, and move the SupportedSocialMedia type and icon to a type file

Reply:
i did use svg but forgot to put this in uicontent